{"name": "test_negative_requests_with_invalid_id[999-4041]", "status": "failed", "statusDetails": {"message": "AssertionError: Expected status code 404, but got 400\nassert 400 == 404\n +  where 400 = <Response [400]>.status_code", "trace": "id = 999, expected_status_code = 404\n\n    @pytest.mark.parametrize(\"id, expected_status_code\", [\n        (\"999\", 404),\n        (\" \", 404),\n        (999, 404)\n    ])\n    def test_negative_requests_with_invalid_id(id, expected_status_code):\n        response = get_request(id)\n        response_data = response.json()\n    \n>       assert response.status_code == expected_status_code, f\"Expected status code {expected_status_code}, but got {response.status_code}\"\nE       AssertionError: Expected status code 404, but got 400\nE       assert 400 == 404\nE        +  where 400 = <Response [400]>.status_code\n\ntests/test_suit_get_item_by_id.py:38: AssertionError"}, "parameters": [{"name": "id", "value": "999"}, {"name": "expected_status_code", "value": "404"}], "start": 1739733949264, "stop": 1739733949793, "uuid": "10f8740e-948b-4e85-929d-2422ef626909", "historyId": "b5bf88aa822c08fc0381405795789181", "testCaseId": "2c86efc81d3ba31cbac87bea2d8dd5f5", "fullName": "tests.test_suit_get_item_by_id#test_negative_requests_with_invalid_id", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_suit_get_item_by_id"}, {"name": "host", "value": "MacBook-Pro-As.local"}, {"name": "thread", "value": "57375-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_suit_get_item_by_id"}]}